examples/lb3-application/README.md

```diff
@@ -337,13 +337,203 @@ $ npm start
 Load [http://localhost:3000/](http://localhost:3000/) on your browser. This will
 load the Express app, with mounted LB3 and LB4 applications.
 
-### Need help?
+## Running LB3 tests from LB4
+
+You can run tests in an LoopBack 3 application from the LoopBack 4 application
+it mounted on with command `npm test`.
+
+We want the LoopBack 3 tests to use the LoopBack 4 server rather than the
+LoopBack 3 application. The following guide shows how to run
```

Comment:
@agnes512 Taking the `CoffeeShopApplication` as an example, when you run the app, it sets up a LB4 rest server and mounts the LB3 app as its component. Therefore all endpoints from the LB3 app are exposed on that server.

And that's why to test those LB3 endpoints, we want to guarantee ^ as well 🙂.

Reply:
> You can run tests in an LoopBack 3 application from the LoopBack 4 application
it mounted on with command `npm test`.

The statement is confusing. It's not clear in which directory the test would be run.

Reply:
> We want the LoopBack 3 tests to use the LoopBack 4 server rather than the
LoopBack 3 application.

Does it use the LoopBack 3 application otherwise?

Reply:
> It's not clear in which directory the test would be run.

@hacksparrow The tests are kept in the LB3 application.

- `lb3-app`
  - `tests` // we want to run the tests here
- `src`
  - `lb4app`
  - `__tests__`

Reply:
@hacksparrow I looked at the doc again, and I believe
> You can run tests in an LoopBack 3 application from the LoopBack 4 application
it mounted on with command `npm test`.

says clearly which directory the test would be run. Or did you mean anything else?